.gitignore

```diff
@@ -4,6 +4,7 @@ logs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+*.tsbuildinfo
```

Comment:
What's this?

Reply:
TypeScript 3.4 introduced the `--incremental` option to enable [faster subsequent builds](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#faster-subsequent-builds-with-the---incremental-flag). This flag tells TS to save information about the project graph in a file called `.tsbuildinfo`. The next time TypeScript is invoked with `--incremental`, it will use that information to detect the least costly way to type-check and emit changes to your project. The incremental option is also enabled by default when you have [composite projects](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#composite-projects). Since we are going monorepo, both of these features can be useful during development and so I added it here.